### PR TITLE
[RHPAM-4439] Vulnerable Version Of PostgreSQL Found

### PR DESCRIPTION
**Pull request by desmax74**
Signed-off-by: desmax74 <mdessi@redhat.com>
See: https://issues.redhat.com/browse/RHPAM-4439

### CODE REVIEW

File: pkg/controller/kieapp/constants/constants.go [hunk 1]
```diff
@@ -51,8 +51,8 @@ var VersionConstants = map[string]*api.VersionConfigs{
 		DatagridComponent:   Datagrid73Component,
 		MySQLImageURL:       MySQL80ImageURL,
 		MySQLComponent:      MySQL80Component,
-		PostgreSQLImageURL:  PostgreSQL10ImageURL,
-		PostgreSQLComponent: PostgreSQL10Component,
+		PostgreSQLImageURL:  PostgreSQL12ImageURL,
```
Comment (spolti) [inline pkg/controller/kieapp/constants/constants.go:54]: see my review on the first PR

**Review by spolti — CHANGES_REQUESTED** [review]

File: pkg/controller/kieapp/constants/constants.go (hunk 1 above)
Comment (desmax74) [reply pkg/controller/kieapp/constants/constants.go]: fixed

File: deploy/cluster_role_binding.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ roleRef:
   kind: ClusterRole
   name: business-automation-operator
 subjects:
-- kind: ServiceAccount
+- apiGroup: rbac.authorization.k8s.io
```
Comment (spolti) [inline deploy/cluster_role_binding.yaml:10]: why this addition here?

**Review by spolti — CHANGES_REQUESTED** [review]

File: deploy/cluster_role_binding.yaml (hunk 2 above)
Comment (desmax74) [reply deploy/cluster_role_binding.yaml]: Was a run with a different operator-sdk version, fixed

**Review by spolti — APPROVED** [review]

**Pull request merged**